packages/design-tokens/src/js/index.ts

```diff
@@ -1,8 +1,14 @@
-export type { Theme } from './themes';
-export { lightTheme } from './themes';
-export { darkTheme } from './themes';
+// DEPRECATED in favor of importing lightTheme and darkTheme
+export { colors } from './colors';
+
+// Brand Color
 export { brandColor } from './brandColor';
+export type { BrandColor } from './brandColor/brandColor.types';
 
-// DEPRECATED in favor of importing theme objects above
-export { colors } from './colors';
+// Themes
+export { lightTheme, darkTheme } from './themes';
+export type { Theme, ThemeColors, ThemeShadows } from './themes/types';
+
+// Typography
 export { typography } from './typography';
+export type { ThemeTypography } from './typography';
```

Comment:
Reordering and adding comments see file for better view https://github.com/MetaMask/metamask-design-system/blob/ad2f94b3a73c932789c8de9f77d8bf3ba0c91c60/packages/design-tokens/src/js/index.ts